src/main/java/frc/robot/subsystems/Elevator.kt

```diff
@@ -57,7 +57,7 @@ object Elevator : SubsystemBase("Elevator") {
                 }
             )
             .andThen(Commands.idle())
-            .until(isDown)
+            .until(isStowed)
```

Comment:
I'm wary of this method existing if the state that it wants to go to isn't a state that will result in `isStowed` being true. We should have some handling of calling this method with an 'up' state